cassandra/src/main/scala/geotrellis/spark/io/cassandra/CassandraLayerWriter.scala

```diff
@@ -33,8 +36,57 @@ class CassandraLayerWriter(
   instance: CassandraInstance,
   keyspace: String,
   table: String
-) extends LayerWriter[LayerId] {
+) extends LayerWriter[LayerId] with LazyLogging {
+
+  // Layer updating
+  def overwrite[
+    K: AvroRecordCodec: Boundable: JsonFormat: ClassTag,
+    V: AvroRecordCodec: ClassTag,
+    M: JsonFormat: GetComponent[?, Bounds[K]]: Mergable
+  ](
+    id: LayerId,
+    rdd: RDD[(K, V)] with Metadata[M]
+  ): Unit = {
+    update(id, rdd, None)
+  }
+
+  def update[
```

Comment:
These `LayerWriter`s seem to have a number of common delegate/convenience methods. Any way they can be abstracted over all/most of the cases?

Reply:
We could expose the overload with `mergeFunc: Option[(V, V) => V]` but its a little iffy because having that as `None` turns a function into overwrite mode, which is drastically more dangerous. I hesitated to that that, do you think it would be helpful though?

Reply:
Not sure which way the thumb is going is it:
- "No, way too dangerous!"
- "Yes, Lets expose the common API"